hive-metastore/src/test/java/org/apache/iceberg/hive/HiveCreateReplaceTableTest.java

```diff
@@ -197,42 +196,41 @@ public void testReplaceTableTxnTableModifiedConcurrently() {
 
     // the replace should still succeed
     table = catalog.loadTable(TABLE_IDENTIFIER);
-    Assert.assertNull("Table props should be updated", table.properties().get("another-prop"));
-    Assert.assertEquals("Table props should match", "value", table.properties().get("prop"));
+    Assertions.assertThat(table.properties().get("another-prop")).as("Table props should be updated").isNull();
+    Assertions.assertThat(table.properties().get("prop")).as("Table props should match").isEqualTo("value");
   }
 
   @Test
   public void testCreateOrReplaceTableTxnTableNotExists() {
-    Assert.assertFalse("Table should not exist", catalog.tableExists(TABLE_IDENTIFIER));
+    Assertions.assertThat(catalog.tableExists(TABLE_IDENTIFIER)).as("Table should not exist").isFalse();
 
     Transaction txn = catalog.newReplaceTableTransaction(TABLE_IDENTIFIER, SCHEMA, SPEC, true);
     txn.updateProperties().set("prop", "value").commit();
     txn.commitTransaction();
 
     Table table = catalog.loadTable(TABLE_IDENTIFIER);
-    Assert.assertEquals("Table props should match", "value", table.properties().get("prop"));
+    Assertions.assertThat( table.properties().get("prop")).as("Table props should match").isEqualTo("value");
```

Comment:
it's ok to add a static import for assertThat (as long as it doesn't cause large unnecessary diffs for existing code)

Reply:
Changed the code to use static import if it's not causing large diffs